tf-modules/internal/cumulus-test-cleanup/iam.tf

```diff
@@ -40,10 +40,21 @@ data "aws_iam_policy_document" "test_cleanup_policy_document" {
     ]
     resources = ["*"]
   }
+  statement {
+    actions = [
+      "s3:ListAllMyBuckets",
+      "s3:GetBucket*",
+      "s3:GetObject*",
+      "s3:ListBucket*",
+      "s3:DeleteObject",
+      "s3:DeleteObjectVersion",
+    ]
+    resources = ["*"]
+  }
 }
 
 resource "aws_iam_role_policy" "test_cleanup_lambda_role_policy" {
-  name   = "${var.prefix}_test_cleanup_lambda_role_policy"
+  name   = "test_cleanup_lambda_role_policy"
```

Comment:
Is this module manually deployed?

Reply:
Yes and it should only be deployed once for the account, otherwise you're running multiple jobs doing the same thing. It's already deployed for our account, although I need to take a pass through SIT.